src/call_builder.js

```diff
@@ -30,6 +30,12 @@ export class CallBuilder {
     }
 
     if (this.filter.length === 1) {
+      this.filter[0].forEach(element => {
+        if (typeof element === 'undefined') {
+          throw new _errors.BadRequestError(`Invalid values are set in filters specified - ${element}`, this.filter);
+        }
+      });
+
```

Comment:
Can you add a unit test that reproduces the failure case you found, and which passes as a result of your change?